docs/spec/data-plane.md

```diff
@@ -91,6 +91,21 @@ delivered.
 CloudEvents received by Sink MAY have
 [Distributed Tracing Extension Attribute](https://github.com/cloudevents/spec/blob/v1.0/extensions/distributed-tracing.md).
 
+### Event reply contract
+
+An event sender supporting event replies SHOULD include a `Prefer: ["callable"]` header
```

Comment:
The prefer header is not an array, it is sent as a single string and you can send more than one prefer header in a single request. The consumer of that request will collate the headers into an array.

Reply:
Make sense. Thanks. Fixed as suggested.